src/operators.jl

```diff
@@ -308,12 +308,14 @@ const _bindings = Dict() # XXX GC Issue? can't use WeakKeyDict with Pairs...
 const _active_binds = Dict()
 
 """
-    `bind!(dest, src, twoway=true)`
+    `bind!(dest, src, twoway=true; initial=true)`
 
 for every update to `src` also update `dest` with the same value and, if
-`twoway` is true, vice-versa.
+`twoway` is true, vice-versa. If `initial` is false, `dest` will only be updated
+to `src`'s value when `src` next updates, otherwise (if `initial` is true) both
```

Comment:
maybe just merge it, not sure I'll confidently do that in before next week ... Thanks for the info!!